### ISSUE TITLE
Noto Sans should be available via the Microsoft Store.

### ISSUE THREAD

**Issue by BEEDELLROKEJULIANLOCKHART** [notofonts/latin-greek-cyrillic#23]

**Comment by simoncozens** [comment]
I think the range of fonts available on the Microsoft Store is up to Microsoft, not to third-party providers. I'll pass on the request to some contacts at Microsoft, but it's kind of out of our control.

**Comment by rokejulianlockhart** [comment]
Yeah, @simoncozens, seems that way, per https://answers.microsoft.com/en-us/windows/forum/all/is-there-a-way-to-sell-fonts-in-the-microsoft/b82dc690-e340-4b4f-b209-46b9a6763c11#:~:text=Report%20abuse-,answer,-Van%20Dennis. Apologies.

Would you inform me of the response when/if you receive one?

**Comment by RokeJulianLockhart** [comment]
This might be more feasible when https://github.com/microsoft/winget-cli/issues/166#issue-619178513 is completed.
